src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -5467,97 +5466,97 @@ class CreateStageSegment(BaseSegment):
                     optional=True,
                 ),
             ),
-            # External S3 stage
```

Comment:
Everything here and below is just a reindenting. I pulled

```
                "URL",
                Ref("EqualsSegment"),
```

up a level instead of repeating it in each case